test/run-test.ps1

```diff
@@ -131,6 +131,15 @@ Get-ChildItem -Recurse -Filter Dockerfile |
                     }
                 }
 
+                if ($sdkTag -like "*2.0-sdk") {
+                    # Temporary workaround https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/dogfooding.md#option-2-self-contained
+                    exec { docker run --rm `
+                        -v ${selfContainedVol}":${containerRoot}volume" `
+                        $selfContainedImage `
+                        chmod u+x ${containerRoot}volume${platformDirSeparator}test
```

Comment:
A Docker volume is being used here versus volume mapping a host directory into the container.  I am not sure how to do this directly so I did it the only way I know how.  Please let me know if you know of a more elegant solution.  Keep in mind this is temporary until the underlying product issue is fixed.